src/tools/chip-cert/Cmd_GenCert.cpp

```diff
@@ -42,6 +42,7 @@ OptionDef gCmdOptionDefs[] =
 {
     { "type",                kArgumentRequired, 't' },
     { "subject-chip-id",     kArgumentRequired, 'i' },
+    { "subject-matter-id",   kArgumentRequired, 'i' },
```

Comment:
maybe a comment saying that aliasing "i" to the same thing is applicable (that `i` reuse for the short name is not a bug).

Reply:
Done.